nncf/quantization/algorithms/min_max/openvino_backend.py

```diff
@@ -45,41 +46,50 @@
 from nncf.scopes import IgnoredScope
 
 
+# pylint:disable=too-many-public-methods
 @ALGO_BACKENDS.register(BackendType.OPENVINO)
 class OVMinMaxAlgoBackend(MinMaxAlgoBackend):
     @property
-    def mat_mul_metatype(self) -> OperatorMetatype:
-        return ov_metatypes.OVMatMulMetatype
+    def mat_mul_metatypes(self) -> List[OperatorMetatype]:
+        return [om.OVMatMulMetatype]
 
     @property
     def post_processing_metatypes(self) -> List[OperatorMetatype]:
-        return [ov_metatypes.OVTopKMetatype, ov_metatypes.OVNonMaxSuppressionMetatype]
+        return [om.OVTopKMetatype, om.OVNonMaxSuppressionMetatype]
 
     @property
     def shapeof_metatypes(self) -> List[OperatorMetatype]:
-        return [ov_metatypes.OVShapeOfMetatype]
+        return [om.OVShapeOfMetatype]
 
     @property
-    def conv_metatype(self) -> List[OperatorMetatype]:
-        return [ov_metatypes.OVConvolutionMetatype]
+    def conv_metatypes(self) -> List[OperatorMetatype]:
+        return [om.OVConvolutionMetatype]
 
     @property
     def overflow_fix_metatypes(self) -> List[OperatorMetatype]:
         return [
-            ov_metatypes.OVConvolutionMetatype,
-            ov_metatypes.OVGroupConvolutionMetatype,
-            ov_metatypes.OVConvolutionBackpropDataMetatype,
-            ov_metatypes.OVGroupConvolutionBackpropDataMetatype,
-            ov_metatypes.OVMatMulMetatype,
+            om.OVConvolutionMetatype,
+            om.OVGroupConvolutionMetatype,
+            om.OVConvolutionBackpropDataMetatype,
+            om.OVGroupConvolutionBackpropDataMetatype,
+            om.OVMatMulMetatype,
         ]
 
     @property
     def read_variable_metatypes(self) -> List[OperatorMetatype]:
-        return [ov_metatypes.OVReadValueMetatype]
+        return [om.OVReadValueMetatype]
+
+    @property
+    def elementwise_metatypes(self) -> List[OperatorMetatype]:
+        return [om.OVAddMetatype]
```

Comment:
I think the method should be renamed, because it contains only Add metatype.
OpenVINO backend has elementwise metatypes list
https://github.com/openvinotoolkit/nncf/blob/34b6b82a746ddac6668c64f012edfe97de090f06/nncf/openvino/hardware/pattern_operations.py#L58-L73

Reply:
Sounds reasonable, thanks. Done.